src/lib/get-css-variable-from-document.ts

```diff
@@ -1,13 +1,37 @@
+interface GetCSSVariableFromDocumentOptions {
+  /**
+   * Whether or not to attempt to coerce the value to an integer
+   */
+  asNumber?: boolean
+
+  /**
+   * An optional fallback value which will be returned if the variable can't be read
+   */
+  fallback?: number | string
+}
+
+/**
+ * Attempt to read a CSS custom property value for use in JS.
+ */
 export default function getCSSVariableFromDocument(
   variableName: string,
-  options: { asNumber?: boolean } = {}
+  options: GetCSSVariableFromDocumentOptions = {}
 ): string | number {
-  const value = getComputedStyle(document.documentElement).getPropertyValue(
-    variableName
-  )
+  let value: string | number = getComputedStyle(
+    document.documentElement
+  ).getPropertyValue(variableName)
 
   if (options.asNumber) {
-    return parseInt(value, 10)
+    value = parseInt(value, 10)
+  }
+
+  if (options.fallback && (value === '' || Number.isNaN(value))) {
+    if (process.env.NODE_ENV === 'development') {
+      console.warn(
+        `Unable to read ${variableName}, falling back to static value.`
+      )
+    }
+    return options.fallback
```

Comment:
⛏️ Should this set `value` instead of early return?

```suggestion
    value = options.fallback
```